test/repository.spec.js

```diff
@@ -240,6 +240,16 @@ describe('Repository', function() {
             done();
          }));
       });
+      
+      it('should list review comments', function(done) {
+        const repo = github.getRepo('michael', 'github');
```

Comment:
This project was moved to the github-tools project. You'll want to update this to get the tests working.